pages/agent/v3/gcloud.md.erb

```diff
@@ -44,18 +44,18 @@ Double check it’s running by taking a look at the cluster info:
 ```shell
 $ kubectl cluster-info
 Kubernetes master is running at https://123.45.67.89
-GLBCDefaultBackend is running at https://123.45.67.89/api/v1/proxy/namespaces/kube-system/services/default-http-backend
-Heapster is running at https://123.45.67.89/api/v1/proxy/namespaces/kube-system/services/heapster
-KubeDNS is running at https://123.45.67.89/api/v1/proxy/namespaces/kube-system/services/kube-dns
-kubernetes-dashboard is running at https://123.45.67.89/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
+GLBCDefaultBackend is running at https://123.45.67.89/api/v1/namespaces/kube-system/services/default-http-backend:http/proxy
+Heapster is running at https://123.45.67.89/api/v1/namespaces/kube-system/services/heapster/proxy
+KubeDNS is running at https://123.45.67.89/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://123.45.67.89/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
```

Comment:
These will vary quite a bit just based on how the user configured their cluster when they created it — dunno if that’s a problem, but it’s pretty likely they could end up with some different stuff here...

Reply:
Ah right! I don't think that's a problem. Hopefully people will let us know if they're finding it confusing.